core/models.py

```diff
@@ -27,11 +27,13 @@ def db_type(self, connection):
         return "timestamp"
 
 
-class Version(models.Model):
-    version = models.TextField(primary_key=True)
+class Constants(models.Model):
```

Comment:
Is this model going to represent both the version and the install ID?  I think we'd need some sort of key + value columns in that case:

```python
version = Constants.objects.get(name="version")
version.value # => "v5.0.1"

install_id = Constants.objects.get(name="install_id")
install_id.value # => some UUID
```

Reply:
Oh now I understand what you meant in your original comment, you want each constant to be a row in the Constants table instead of having a model describing what constants we have,  I will modify this PR to do that